### PR TITLE
[3006][ADD] report_qweb_decimal_place

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3006](https://www.quartile.co/web#id=3006&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Review by kakurai8 — COMMENTED** [review]
I checked the current status.
(Settings is added, but this module alone doesn't work.)
report_qweb_decimal_place

**Comment by AungKoKoLin1997** [comment]
@kakurai8 This module alone will not work. This module is for base configuration and we need to use with other modules(sale_report_qweb_decimal_places, etcs). I am waiting Yoshi's approval and will continue working on another modules.

File: report_qweb_decimal_place/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    >
+        <xpath expr="//td[@name='td_priceunit']/span" position="replace">
+                <t t-set="currency" t-value="doc.currency_id" />
+                <t t-set="price_unit" t-value="line.price_unit" />
+                <t t-call="report_qweb_decimal_place.price_unit_value_format" />
+        </xpath>
+    </template>
+
+Background:
+-----------
```
Comment (yostashiro) [inline report_qweb_decimal_place/readme/DESCRIPTION.rst]:
```suggestion
~~~~~~~~~~~

```
See https://github.com/OCA/maintainer-tools/pull/455#issuecomment-636305924.

Please update the OCA PR as well.

**Review by yostashiro — CHANGES_REQUESTED** [review]

File: report_qweb_decimal_place/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    >
+        <xpath expr="//td[@name='td_priceunit']/span" position="replace">
+                <t t-set="currency" t-value="doc.currency_id" />
+                <t t-set="price_unit" t-value="line.price_unit" />
+                <t t-call="report_qweb_decimal_place.price_unit_value_format" />
+        </xpath>
+    </template>
+
+Background:
+~~~~~~~~~~~
```
Comment (yostashiro) [inline report_qweb_decimal_place/readme/DESCRIPTION.rst:20]:
```suggestion
~~~~~~~~~~~

```
Please pay attention to the suggested blank line!

**Review by yostashiro — CHANGES_REQUESTED** [review]

File: report_qweb_decimal_place/README.rst [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To apply price unit format:
+ - Go to Invoicing --> Configuration --> Currencies
+ - Check apply_price_decimal_place field
+ - Define decimal place in price_decimal_place field
```
Comment (yostashiro) [inline report_qweb_decimal_place/README.rst]:
```suggestion
To apply price unit format:

#. Go to Invoicing --> Configuration --> Currencies
#. Check apply_price_decimal_place field
#. Define decimal place in price_decimal_place field
```

**Review by yostashiro — APPROVED** [review]

**Comment by yostashiro** [comment]
This module should be added in fal-oca.

**Comment by yostashiro** [comment]
Superseded by https://github.com/qrtl/fal-oca/pull/9.

**Pull request closed without merging**